### PR TITLE
Classification tests

### PR DESCRIPTION
**Pull request by carpecodeum**
# Description

provided tests to classification route

Fixes # (issue)152

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Screenshot 2019-12-27 at 4 50 41 AM

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by carpecodeum** [comment]
@Tushar19varshney please review

**Comment by carpecodeum** [comment]
> @carpecodeum Just reviewed your PR, It would be great if you don't repeat yourself in writing the same code more than once, I think you should create a helper folder and write all the helping functions like connecting to mongo and creating user and login should be there and then it can be used anywhere in the test just by importing.
> Also please check your branch log, since there are lots of commit of other PR as well.
> Otherwise everything else looks great to me.
> Great work @carpecodeum !

hey, @Rupeshiya yes i definitely have plans for that and minimizing writing the same code over and over and thats why i have created a sample dump file where all the data regarding the tests and the helpers functions will reside this just needs approval from one of the maintainers so that i can go ahead with this idea.

File: labellab-server/config/dbURI.js [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
 var dbCluster = process.env.DB_CLUSTER
 var dbUser = process.env.DB_USERNAME
 var dbPass = process.env.DB_PASSWORD
+var dbPort = process.env.DB_PORT
 
 let mongoURI
 
 if(dbName && dbUser && dbPass){
-	mongoURI = '' + dbHost + '://'+ dbUser + ':' + dbPass + '@' + dbCluster + '/' + dbName
+	mongoURI = 'mongodb+srv://adi:adi9891255973@adi-ocz5j.mongodb.net/test?retryWrites=true&w=majority'
```
Comment (Tushar19varshney) [inline labellab-server/config/dbURI.js]: We have created `.env` for saving credentials, you don't have to hard code your credentials.

**Comment by carpecodeum** [comment]
closing this PR and opening a new PR for cleaner and specific commit history

**Pull request closed without merging**